locales/en/common.json

```diff
@@ -52,6 +52,17 @@
   "LANG_SWITCHER": "日本語",
   "COPYRIGHT_LINE": "© 2022 henkaku.org. All rights reserved.",
 
+  "claim": {
+    "toast": {
+      "submitted": "Submitted",
+      "error": "Error",
+      "success": "You got $HENKAKU.",
+      "insufficientAmount": "You don't have points to claim $HENKAKU.",
+      "insufficientFound": "Currently unable to claim.",
+      "txFailed": "Transaction failed.",
+      "defaultError": "Unknown Server Error"
+    }
+  },
```

Comment:
To be moved to the claim namespace after https://github.com/henkaku-center/omise-interface/pull/114 is merged